src/components/AddItemForm.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
+import fb from '../lib/firebase';
+import '../css/AddItemForm.css';
+import { v4 as uuidv4 } from 'uuid';
+
+const Form = () => {
+
+    const [itemName, setName] = useState("");
+    const [timeFrame, setTimeFrame] = useState(0);
+    const [lastPurchasDate, setDate] = useState(null);
+    // const userToken = token || "testToken"
+
+    const handleSubmit = e => {
+        e.preventDefault();
+        let db = fb.firestore();
+        let data = {
+            id: uuidv4(),
+            itemName,
+            timeFrame: parseInt(timeFrame),
+            lastPurchasDate
+        };
+        console.log(data);
+        db.collection("testToken").add(data);
+    }
+
+  return (
+	<div>
+
+    <form onSubmit={e => handleSubmit(e)}>
+
+        <div><h1>Name of the item</h1></div>
+        <input
+            name="name"
```

Comment:
You did a great job with native form fields. Next time (don't change it for now), I would go with a form label rather than an `h1` to describe the input.